### ISSUE TITLE
Lost connection

### ISSUE THREAD

**Issue by normen** [homebridge-xiaomi-roborock-vacuum/homebridge-xiaomi-roborock-vacuum#135]
Sometimes the plugin seems to lose connection to the Roborock. When that happens the Roborock is accessible through the Xiaomi Home App and there is no error messages in the HomeBridge log. The plugin still seems to report states from the Roborock in the log when that happens as well:
```
[2020-3-15 15:29:23] [Staubsauger] INF getCleaning | roborock.vacuum.s5 | Cleaning is true
[2020-3-15 15:29:23] [Staubsauger] INF getCleaning | roborock.vacuum.s5 | Cleaning is true
[2020-3-15 15:29:23] [Staubsauger] INF getBatteryLow | roborock.vacuum.s5 | Batterylevel is 100%
[2020-3-15 15:29:23] [Staubsauger] INF getCleaning | roborock.vacuum.s5 | Cleaning is true
```
In this case I gave the command to start a room cleaning, the HomeBridge log says the state is changing to "Cleaning" but the Roborock doesn't do anything.

Restarting the homebridge server solves the issue. I can't reliably reproduce it.

Cheers for this plugin,
Normen

**Comment by afharo** [comment]
Hi @normen, can you run this in `debug` mode so we can see what messages are going through in the communication with the robot? This way we can better analyse the issue?

**Comment by normen** [comment]
It only happened again once until now, I'll set it to debug mode and try if I can catch the issue. I usually only notice after a while so I can't make any promises that the messages won't be lost in the log rotation.

**Comment by afharo** [comment]
Thank you! 

It might be related to [this comment](https://github.com/nicoh88/homebridge-xiaomi-roborock-vacuum/issues/146#issuecomment-622924762). But it's best to have more info.

**Comment by bedrin** [comment]
Probably fixed by https://github.com/homebridge-xiaomi-roborock-vacuum/homebridge-xiaomi-roborock-vacuum/pull/194

**Comment by afharo** [comment]
Published as `homebridge-xiaomi-roborock-vacuum@0.9.0-3`. If confirmed all is good, I'll publish the Final release `0.9.0`. Thank you @bedrin
